src/core/parser.ml

```diff
@@ -655,9 +660,10 @@ let parser cmd =
   | _open_ ps:path+
       -> List.iter (get_ops _loc) ps;
          P_open(ps)
-  | e:exposition? p:property? _symbol_ s:ident al:arg* ":" a:term
+  | st:mstrat? e:exposition? p:property? _symbol_ s:ident al:arg* ":" a:term
```

Comment:
Could we simplify the grammar (for the user) so that she doesn't need to remember in which order to put all these modifiers? For instance using:
```
(* in syntax.ml *)
type p_modifier = Stra of match_strat | Prop of property | Expo of exposition
```
A symbol declaration would come with a list of modifiers.
Of course, we would then need to check that the list is consistent: fail when there is both protected and private, or constant and injective.